app/Resources/translations/messages.en_GB.xliff

```diff
@@ -40,52 +42,62 @@
       <trans-unit id="b4adb3d9176c2fbdf293bee1b7dda2c6fb56c67c" resname="ss.flash.error_while_switching_locale">
         <source>ss.flash.error_while_switching_locale</source>
         <target>Due to an unknown reason, switching locales failed.</target>
-        <jms:reference-file line="72">/../src/Surfnet/StepupSelfService/SelfServiceBundle/Controller/LocaleController.php</jms:reference-file>
+        <jms:reference-file line="73">/../src/Surfnet/StepupSelfService/SelfServiceBundle/Controller/LocaleController.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="600028302c7b3a43ae0e30ab3b015f505c2b7f1f" resname="ss.flash.invalid_switch_locale_form">
         <source>ss.flash.invalid_switch_locale_form</source>
         <target>Due to an unknown reason, switching locales failed.</target>
-        <jms:reference-file line="65">/../src/Surfnet/StepupSelfService/SelfServiceBundle/Controller/LocaleController.php</jms:reference-file>
+        <jms:reference-file line="66">/../src/Surfnet/StepupSelfService/SelfServiceBundle/Controller/LocaleController.php</jms:reference-file>
+      </trans-unit>
+      <trans-unit id="0ba164e8499593f2b376e03b4fb3483a41696c7f" resname="ss.form.ss_remote_vet_second_factor.cancel">
+        <source>ss.form.ss_remote_vet_second_factor.cancel</source>
+        <target state="new">ss.form.ss_remote_vet_second_factor.cancel</target>
```

Comment:
I think it will be betterl do this later on because there have to be an intermediate step to select the method to vet (face2face/remote). Then this probably wouldn't be needed after all.